flow-typed/box-annotations.js

```diff
@@ -138,4 +140,20 @@ type AnnotationData = {
 };
 
 type StringAnyMap = { [string]: any };
-type AnnotationMap = { [string]: AnnotationData };
\ No newline at end of file
+type AnnotationMap = { [string]: AnnotationData };
+
+type AnnotationThread = {
+    annotatedElement: HTMLElement,
+    annotations: Array<AnnotationData>,
+    api: AnnotationAPI,
```

Comment:
Would it make sense to decouple the API from the `AnnotationThread` class? Maybe introduce an annotation API service that you can call into to perform the API calls?

Reply:
That's the plan in a subsequent PR. This is leg work for that